.ddev/.disabled-services/docker-compose.chromedriver.yaml

```diff
@@ -6,13 +6,24 @@
 version: '3.6'
 services:
   chromedriver:
-    image:  previousnext/chrome-headless:65
+    image: seleniarm/standalone-chromium:4.1.4-20220429
+    shm_size: 2gb
```

Comment:
This is needed to prevent ERR_INSUFFICIENT_RESOURCES in Chrome, but I'm not sure why we don't have it specified in https://github.com/Lullabot/drainpipe-dev/blob/main/scaffold/nightwatch/docker-compose.selenium.yaml. I want to followup on that.

Reply:
Ah-ha! This is now not recommended. In drainpipe, we disable shared memory via this flag: https://developers.google.com/web/tools/puppeteer/troubleshooting#tips

I'll find out about getting this flipped. I'm curious why it's not by default in Selenium's containers.

Reply:
According to https://github.com/puppeteer/puppeteer/issues/1834 it's not default because it's not a "solve everywhere" problem. Since you're storing to disk, it's entirely possible to run out of disk space, or have much slower tests.

It looks like Cypress has landed on disabling this by default, so I'm guessing in practice it's better to "assume Docker": https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/browsers/chrome.ts#L118-L120

However, locally I'm seeing around a 30% performance hit by disabling the flag. Perhaps we're happier to just set a limit? I'm going to do a little more benchmarking, because adding minutes to test runs seems worth it.

Reply:
I tested with both on and off with:

`time (for i in {1..3}; do vendor/bin/phpunit --testsuite existing-site-javascript --filter testWideAlertPlacement; done)`

Test times were not noticeably different, taking around 90s each. I'll push up a commit taking upstream's recommendation.